src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -121,9 +121,9 @@ class Docker implements Serializable {
             }
         }
 
-        public Container run(String args = '') {
+        public Container run(String args = '', String command = "") {
             docker.node {
-                docker.script.sh "docker run -d${args != '' ? ' ' + args : ''} ${id} > .container"
+                docker.script.sh "docker run -d${args != '' ? ' ' + args : ''} ${id} ${command} > .container"
```

Comment:
Suggest using the same trick as with `args` to avoid inserting a gratuitous space into the generated command in the common case that `command` is not specified.